pkg/integration/zsh.go

```diff
@@ -0,0 +1,6 @@
+package integration
+
+var zshSource = `
+prmptcmd() { $("__dad_prompt_command") }
```

Comment:
Do we need to call the prompt command in a subshell with `$(...)` ?
Could we just call it directly? `prmptcmd() { __dad_prompt_command }`

Reply:
`promptcmd`? (with a `o`)

Reply:
Yes sadly it seems it needs to be called in a subshell cause if not, it would run as soon as you type `prmptcmd() { __dad_prompt_command }` and not everytime a prompt is being displayed